.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: hhvm
+    - php: nightly
```

Comment:
using 7.0 would be better than nightly, as we will hopefully have 7.0 releases available on Travis in their next environment update (for now, it also installs the nightly) and nightly would become 7.1 once the PHP team creates a separate branch